<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Webcam viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="lblWebcam">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>500</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>500</width>
        <height>500</height>
       </size>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="exitButton">
      <property name="text">
       <string>EXIT</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="4" column="1">
       <widget class="QPushButton" name="upButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>UP</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="stopButtonKC">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>STOP</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="downButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>DOWN</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="leftButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>LEFT</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="rightButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>RIGHT</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="connectButton">
      <property name="text">
       <string>CONNECT</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="disconnectButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>DISCONNECT</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>649</x>
     <y>528</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
